vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/main/resources/META-INF/resources/frontend/menubarConnector.js

```diff
@@ -73,8 +73,14 @@ import './contextMenuConnector.js';
 
         let items = menubar.__generatedItems || [];
 
-        // Propagate disabled state from items to parent buttons
-        items.forEach((item) => (item.disabled = item.component.disabled));
+        items.forEach((item) => {
+          // Propagate disabled state from items to parent buttons
+          item.disabled = item.component.disabled;
+
+          // Saving item to component because `_item` can be reassigned to a new value
+          // when the component goes to the overflow menu
+          item.component._rootItem = item;
```

Comment:
I always find it confusing. What does item represent here? Is it an HTML element?